.gitignore

```diff
@@ -1 +1,5 @@
 docs/_site
+
+draco.vcxproj
```

Comment:
Can the rest of the RTC ignores be added. Preferably like this https://github.com/Esri/skia/blob/runtimecore/.gitignore#L59